packages/dx-react-chart/src/plugins/area-series.test.jsx

```diff
@@ -14,6 +14,7 @@ jest.mock('@devexpress/dx-chart-core', () => ({
   coordinates: jest.fn(),
   seriesData: jest.fn(),
   checkZeroStart: jest.fn(),
+  getItemsCallback: jest.fn,
```

Comment:
Why we use `jest.fn` instead of `jest.fn()` here?

Reply:
That's my fault. I'll fix it